plugins/extra/p2js/componentConfigs/P2BodyConfig.ts

```diff
@@ -55,6 +58,10 @@ export default class P2BodyConfig extends SupCore.Data.Base.ComponentConfig {
       if (pub.shape === "rectangle") pub.shape = "box";
     }
 
+    if (pub.formatVersion === 1) {
+      pub.angle = 0;
```

Comment:
Like above, we need to update the formatVersion. So we should have the following line:
`pub.formatVersion = 2;`

Usually, we put this line at the top of the condition.